### ISSUE TITLE
Day selection not accessible with keyboard in Edge

### ISSUE THREAD

**Issue by terjela** [airbnb/react-dates#1502]
**react-dates version**
react-dates@18.3.1

**Describe the bug**
It is not possible to navigate the day selection using the keyboard in Edge. 

**Screenshots/Gifs**
![singledatepicker](https://user-images.githubusercontent.com/8427100/50889916-6bed2980-13f9-11e9-8b2c-11595ca520aa.gif)

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser Edge 
 - Version 42.17134.1.0

**Is the issue reproducible in Storybook?**
[SingleDatePicker](http://airbnb.io/react-dates/?selectedKind=SingleDatePicker%20%28SDP%29&selectedStory=default&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel)

**Comment by majapw** [comment]
I will take a look at this this week. That's a pretty serious bug! D:

**Comment by GithubGoldMiner** [comment]
@majapw Hi, I am just curious, since I am also using this library in my side project. I do realize that Keyboard navigation is not functional as well (Chrome 69 on Win10 machine). Can you confirm with that?

**Comment by eiriks** [comment]
@majapw Hey. Did you get around to take a look at this?

**Comment by backwardok** [comment]
I tested this on Edge on a Windows machine in the latest version, and am not seeing the issue anymore. I also noticed that in 18.4.1, there was a change to make the inputs tabbable (https://github.com/airbnb/react-dates/pull/1499). There is a flicker issue (https://github.com/airbnb/react-dates/issues/1523) that that PR introduced, with an open PR (https://github.com/airbnb/react-dates/pull/1553) to address that. Would recommend upgrading to the latest version (or waiting until that PR gets merged and a new release gets out if you are holding out on the flicker behavior) to address this issue.
